Api/Controllers/RestaurantsController.cs

```diff
@@ -226,34 +227,86 @@ public async Task<ActionResult<RestaurantVM>> GetRestaurantDetails(int restauran
         return Ok(result.Value);
     }
 
-        /// <summary>
-        /// Get visits in a restaurant
-        /// </summary>
-        /// <param name="restaurantId">ID of the restaurant.</param>
-        /// <param name="dateStart">Filter out visits before the date</param>
-        /// <param name="dateEnd">Filter out visits ater the date</param>
-        /// <param name="visitSorting">Order visits</param>
-        /// <param name="page">Page number</param>
-        /// <param name="perPage">Items per page</param>
-        /// <returns>Paged list of visits</returns>
-        [HttpGet("{restaurantId:int}/visits")]
-        [ProducesResponseType(200), ProducesResponseType(400)]
-        public async Task<ActionResult<Pagination<VisitVM>>> GetVisitsInRestaurant(
-            int restaurantId,
-            DateOnly? dateStart,
-            DateOnly? dateEnd,
-            VisitSorting visitSorting,
-            [FromQuery] int page = 0,
-            [FromQuery] int perPage = 10)
+    /// <summary>
+    /// Get visits in a restaurant
+    /// </summary>
+    /// <param name="restaurantId">ID of the restaurant.</param>
+    /// <param name="dateStart">Filter out visits before the date</param>
+    /// <param name="dateEnd">Filter out visits ater the date</param>
+    /// <param name="visitSorting">Order visits</param>
+    /// <param name="page">Page number</param>
+    /// <param name="perPage">Items per page</param>
+    /// <returns>Paged list of visits</returns>
+    [HttpGet("{restaurantId:int}/visits")]
+    [ProducesResponseType(200), ProducesResponseType(400)]
+    public async Task<ActionResult<Pagination<VisitVM>>> GetVisitsInRestaurant(
+        int restaurantId,
+        DateOnly? dateStart,
+        DateOnly? dateEnd,
+        VisitSorting visitSorting,
+        [FromQuery] int page = 0,
+        [FromQuery] int perPage = 10)
+    {
+        var result = await service.GetVisitsInRestaurantAsync(restaurantId, dateStart, dateEnd, visitSorting, page, perPage);
+
+        if (result.IsError)
         {
-            var result = await service.GetVisitsInRestaurantAsync(restaurantId, dateStart, dateEnd, visitSorting, page, perPage);
+            return result.ToValidationProblem();
+        }
+
+        return Ok(result.Value);
+    }
+
+
+
 
-            if (result.IsError)
-            {
-                return result.ToValidationProblem();
-            }
 
-            return Ok(result.Value);
+
+    /// <summary>
+    /// Get list of menus by given restaurant id
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("{restaurantId:int}/menus")]
+    [ProducesResponseType(200)]
+    [ProducesResponseType(404)]
+    [Authorize(Roles = Roles.Customer)]
+    public async Task<ActionResult<List<MenuSummaryVM>>> GetMenusById(int restaurantId)
```

Comment:
Ta końcówka jest starsza od podejścia z Result, skoro już jesteśmy tu, to można dorobić kody błędów tak jak w GetMenuItems